io.github.loot.loot.yml

```diff
@@ -0,0 +1,51 @@
+app-id: io.github.loot.loot
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: LOOT
+modules:
+  - name: application
+    buildsystem: simple
+    build-commands:
+      - install -D bin/LOOT /app/bin/LOOT
+      - install -D lib/libloot.so /app/lib/libloot.so
+      - cp -r share /app
+    sources:
+      - type: archive
+        url: https://github.com/loot/loot/releases/download/0.21.0/loot_0.21.0-Linux.tar.xz
+        sha256: 05e6a4eb1cbb237f507dc39957fe6ac485b7f5f402d351caa1a5fe7dc179c869
+  - name: libtbb
+    buildsystem: simple
+    build-commands:
+      - install -D lib/intel64/gcc4.8/libtbb.so.2 /app/lib/
+      - install -D LICENSE /app/share/doc/libtbb2/copyright
+    sources:
+      - type: archive
+        url: https://github.com/oneapi-src/oneTBB/releases/download/v2020.3/tbb-2020.3-lin.tgz
+        sha256: bb8cddd0277605d3ee7f4e19b138c983f298d69fcbb585385b59ef7239d5ef83
+
+finish-args:
```

Comment:
finish-args should be above modules per convention

Reply:
OK, I've swapped them.